docs/start/getting-started/fragments/android/setup.md

```diff
@@ -45,9 +45,14 @@ Android Studio will open your project with a tab opened to either *MainActivity.
 
 Amplify for Android is distributed as an Apache Maven package. In this section, you'll add the packages and other required directives to your build configuration.
 
-1. Expand **Gradle Scripts** and open **build.gradle (Project: Todo)**.
+1. Expand **Gradle Scripts** in the project file viewer and open **build.gradle (Project: Todo)**.
 
-  Add the following lines:
+  Add the following lines to the file:
```

Comment:
```suggestion
  Make the following additions to the project-level `build.gradle` file:
```

^ Note, there are two important `build.gradle` files. A common source of confusion for beginning Android developers is to understand _which one_ to use, and when.